commit 5da742c6fcaf9f15d4dc5dfba62ec42b59cbfcce
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Tue Feb 7 01:49:45 2017 +0100

    Add ki18n_install and kdoctools_install calls for release tarball

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b053562..d9df8ec 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,6 +70,13 @@ add_subdirectory(smb4k)
 add_subdirectory(data)
 add_subdirectory(doc)
 
+if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/po")
+  ki18n_install(po)
+  if(KF5DocTools_FOUND)
+    kdoctools_install(po)
+  endif(KF5DocTools_FOUND)
+endif(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/po")
+
 ########### Plasmoid ####################
 
 # option(INSTALL_PLASMOID "Install plasmoid" ON)
